includes/discovery/functions.inc.php

```diff
@@ -1248,10 +1248,10 @@ function build_cbgp_peers($device, $peer, $af_data, $peer2)
 
         $afisafi_tmp = explode('.', $k);
         if ($device['os_group'] === 'vrp') {
-            array_shift($afisafi_tmp); //remove 1st value, always 0 so far
+            $vpninst_id  = array_shift($afisafi_tmp);
```

Comment:
This variable is not used (yet) but will be as soon as this vpninst_id is stored properly.